docs/self-hosted-apps/reverseproxy.md

```diff
@@ -0,0 +1,29 @@
+# Self Hosted Reverse Proxy
+
```

Comment:
Can you add some explanation about what is a reverse proxy and why should you use one?